dev/pr/gcs-wif-pipelinerun.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  generateName: pipelinerun-gcs-
+spec:
+  pipelineRef:
+    name: pipeline-wif
+  params:
+    - name: repo_url
+      value: https://github.com/chmouel/go-helloworld
+    - name: revision
+      value: main
+      # This uses GCS bucket  to upload Caches
+    - name: registry
+      value: gs://tekton-caches
+    - name: buildCommand
+      value: go build -v ./
+    - name: image
+      value: golang:1.21
+  workspaces:
+    - name: source
+      emptyDir: { }
+    - name: cred
+      secret:
+        secretName: gcs-cred
+    - name: token
+      projected:
+        sources:
+          - serviceAccountToken:
+              audience: https://iam.googleapis.com/projects/272779626560/locations/global/workloadIdentityPools/openshift-pool/providers/opeshift-wif
```

Comment:
Can you please put this in conventional format for k8s and OpenShift?

Reply:
Some of our customers will complain that WIF isn't properly supported if do this. We already have a mail on this.

Reply:
They way it was setup audience is getting generated like this only. Will check if this can be simplified

Reply:
Yes, that and location where we project should also be similar.